### PR TITLE
labels: Add wg/ipam group label

### PR DESCRIPTION
**Pull request by oshoval**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Checklist**

This checklist is not enforcing, but it's a reminder of items that could be relevant to every PR.
Approvers are expected to review this list.

- [ ] Design: A [design document](https://github.com/kubevirt/community/tree/main/design-proposals) was considered and is present (link) or not required
- [ ] PR: The PR description is expressive enough and will help future contributors
- [ ] Code: [Write code that humans can understand](https://en.wikiquote.org/wiki/Martin_Fowler#code-for-humans) and [Keep it simple](https://en.wikipedia.org/wiki/KISS_principle)
- [ ] Refactor: You have [left the code cleaner than you found it (Boy Scout Rule)](https://learning.oreilly.com/library/view/97-things-every/9780596809515/ch08.html)
- [ ] Upgrade: Impact of this change on upgrade flows was considered and addressed if required
- [ ] Testing: New code requires [new unit tests](https://github.com/kubevirt/kubevirt/blob/main/docs/reviewer-guide.md#when-is-a-pr-good-enough). New features and bug fixes require at least on e2e test
- [ ] Documentation: A [user-guide update](https://github.com/kubevirt/user-guide/) was considered and is present (link) or not required. You want a user-guide update if it's a user facing feature / API change.
- [ ] Community: Announcement to [kubevirt-dev](https://groups.google.com/g/kubevirt-dev/) was considered

**Release note**:

```release-note

```

### CODE REVIEW

**Comment by oshoval** [comment]
/cc @enp0s3 @maiqueb @qinqon @brianmcarey 

can you please take a look ? 
requested by Igor

**Comment by fabiand** [comment]
Folks, I'm really wondering what is going on.

How the heck did we end up with a sig-wg-* prefix?
Who defined the semantics?
Can everybody invent sigs?

PR's like this require us to think about how we formalize - to add more prorcesses that nobody wants - the introductoin of sigs - to define ownership in KubeVirt.

Again: SIGs to define ownership of non-overlapping areas within KubeVirt.
A SIG can then decide on itself how to divide it's area among members. One idea is to use wg-*.

**Comment by oshoval** [comment]
Np thanks, it was draft and i raised a question on slack to understand what name to use
will change to wg-ipam

**Comment by oshoval** [comment]
Np thanks, it was draft and i raised a question on slack to understand what name to use
Do you mean sig-network-wg-ipam ? or just wg-ipam ?

**Comment by fabiand** [comment]
Maybe I was a little bit harsh, I see now it's `sig/wg-ipam`, thus not `sig-wg-ipam`.

We can also not just define `wg-ipam` - it must be associated to a SIG. How do we do this?

Once it is solved, then please use `wg/ipam` instead of `sig/wg-ipam`.

**Comment by fabiand** [comment]
It feels like `sig/network` plus a new label `wg/ipam` could indicate the right thing:

Owned by sig network, handled by wg ipam.

**Comment by oshoval** [comment]
hmm, not sure i did correct, will need to understand better how to do what you suggested
atm changed to sig/network-wg-ipam

**Comment by oshoval** [comment]
updated the label to wg/ipam
hope i understood you correct (once i do, i will update the owners file in the other PR with all the required changes)
thanks

**Comment by enp0s3** [comment]
> Folks, I'm really wondering what is going on.
> 
> How the heck did we end up with a sig-wg-* prefix? Who defined the semantics? Can everybody invent sigs?
> 
> PR's like this require us to think about how we formalize - to add more prorcesses that nobody wants - the introductoin of sigs - to define ownership in KubeVirt.
> 
> Again: SIGs to define ownership of non-overlapping areas within KubeVirt. A SIG can then decide on itself how to divide it's area among members. One idea is to use wg-*.

@fabiand @vladikr This PR just reflects an attempt to comply with what was discussed on Monday's approvres meeting.
IIUC there was a push back from having individual approvers in specific packages, thus the idea with WG came up.

Now to be more specific, with regards to [IPAM PR](https://github.com/kubevirt/kubevirt/pull/11410)
The situation is simple - ex sig-network maintainers are willing to maintain IPAM package, while current sig-network maintainers don't. 

If the WG procedure is yet defined, can we proceed with having individual approvers in `pkg/libipam/OWNERS`?

File: docs/labels.md [hunk 1]
```diff
@@ -84,6 +84,7 @@ larger set of contributors to apply/remove them.
 | <a id="triage/needs-information" href="#triage/needs-information">`triage/needs-information`</a> | Indicates an issue needs more information in order to work on it.| anyone | |
 | <a id="triage/not-reproducible" href="#triage/not-reproducible">`triage/not-reproducible`</a> | Indicates an issue can not be reproduced as described.| anyone | |
 | <a id="triage/unresolved" href="#triage/unresolved">`triage/unresolved`</a> | Indicates an issue that can not or will not be resolved.| anyone | |
+| <a id="wg/ipam" href="#wg/ipam">`wg/ipam`</a> | | anyone |  [label](https://prow.ci.kubevirt.io/command-help#label) |
```
Comment (fabiand) [inline docs/labels.md:87]: In this comment, please sync to sig-network charter in community repo. In sig network charter, please add wg-ipam.

**Comment by fabiand** [comment]
@enp0s3 understood.

My main concern was abut the usage of the string `sig-wg-iapm` (which was a misunderstanding on my side).

As stated above:
sigs own code.
wg are internal structures to delegate who is handling it.


thus we 
a) need to make the connection between wg ipam and sig network
b) sig network remains to own the code/area, wg ipam will handle that specific piece and has trust by sig network to even approve it.

File: docs/labels.md (hunk 1 above)
Comment (oshoval) [reply docs/labels.md]: updated this PR (hope i understood correct)
and created https://github.com/kubevirt/community/pull/300

**Comment by fabiand** [comment]
@oshoval thanks for your openess! and the quick turnaround

The PR including the IMPORTANT links (for user context) to community and label PRs are good.

/unhold

/lgtm

**Comment by dhiller** [comment]
All, sorry for not being able to bring the PR around the initial `wg`s for `arch` to life soon enough: https://github.com/kubevirt/community/pull/298

/approve

Thanks @oshoval

**Pull request merged**